### ISSUE TITLE
RSS feed malformed

### ISSUE THREAD

**Issue by tor255** [omarroth/invidious#270]
According to ruby's `RSS` module, the atom feed for my subscriptions is malformed, giving me the error:
```
This is not well formed XML (RSS::NotWellFormedError)`
Undefined prefix yt found
```

Also, python's `feedparser` also returns an error, giving me this:
`feedparser.UndeclaredNamespace: 'yt' is not associated with a namespace`

Which, I assume, is from:
```xml
<entry>
  <id>yt:video:dQw4w9WgXcQ</id>
  <yt:videoId>dQw4w9WgXcQ</yt:videoId>
```

**Comment by tor255** [comment]
Well, it works fine in python, but I think I figured out what was causing the problem.

In line 2 of feed.xml (`<feed xmlns="http://www.w3.org/2005/Atom" xmlns:media="http://search.yahoo.com/mrss/" xml:lang="en-US">`)

`xmlns="http://www.w3.org/2005/Atom"`
should be
`xmlns:yt="http://www.w3.org/2005/Atom"`

**Comment by omarroth** [comment]
Should be fixed with #271.

**Comment by omarroth** [comment]
This snippet works as expected:
```ruby
require 'rss'

rss = RSS::Parser.parse('https://invidio.us/feed/private?token=PRIVATE_TOKEN', false)
rss.items.each do |item|
  puts item.title
end
```

So I'm going to close this. If any other programs/libraries have trouble with feeds provided by Invidious, feel free to open a new issue.
